pyproject.toml

```diff
@@ -50,9 +50,7 @@ Home-page = "https://github.com/pypa/readme_renderer"
 requires = ["setuptools>=40.8.0"]
 build-backend = "setuptools.build_meta"
 
-# TODO: Remove when https://github.com/mgedmin/check-manifest/pull/155 released
 [tool.check-manifest]
```

Comment:
I think now that there's no need for the rule, is there any need for a config section?
```suggestion
```